<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TW2</class>
 <widget class="QDialog" name="TW2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>269</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TW2</string>
  </property>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>160</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>条指令</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbt_count">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>210</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>计算</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>请  选  择</string>
   </property>
   <property name="buddy">
    <cstring>comboBox_year</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>120</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>美元</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_year">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>40</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>1985</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1995</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pbt_close">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>210</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>程序员平均工资</string>
   </property>
   <property name="buddy">
    <cstring>lineEdit_salary</cstring>
   </property>
  </widget>
  <widget class="QPushButton" name="pbt_clear">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>210</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>清空</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>40</y>
     <width>16</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>年</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_salary">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>120</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_codeline">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>160</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>10</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>30</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>80</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>位</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>计 算 机 字 长</string>
   </property>
   <property name="buddy">
    <cstring>lineEdit_bit</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>程序员每天开发</string>
   </property>
   <property name="buddy">
    <cstring>comboBox_codeline</cstring>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_bit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>80</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>30</y>
     <width>241</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>结果</string>
   </property>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>计算机储存容量需求</string>
    </property>
    <property name="buddy">
     <cstring>lineEdit_need</cstring>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_need">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>储 存 器 价 格</string>
    </property>
    <property name="buddy">
     <cstring>lineEdit_price</cstring>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_price">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>70</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>70</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>美元</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>装满储存器成本</string>
    </property>
    <property name="buddy">
     <cstring>lineEdit_cost</cstring>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_cost">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>110</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>110</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>美元</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pbt_close</sender>
   <signal>clicked()</signal>
   <receiver>TW2</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>345</x>
     <y>221</y>
    </hint>
    <hint type="destinationlabel">
     <x>259</x>
     <y>134</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
